build/Versions.props

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Properties are commented out where they match the current default specified in Repo Toolset -->
```

Comment:
Can we just remove the commented out parts and put a link to the Repo toolset default file. I think it could be misleading if Repo toolset defaults change.